src/components/forms/formComponents.js

```diff
@@ -435,3 +487,46 @@ export const formInputSlider = (config) => {
   ]);
 };
 
+export const formInputFile = (config) => {
+  const {
+    id,
+    uploadText = "Upload a file",
+    multiple = false,
+    accept = "",
+  } = config;
+
+  return div({}, [
+    div({
+      className: 'form-file-upload',
+    }, [
+      input({
+        id,
+        type: 'file',
+        multiple,
+        accept,
+        // make hidden:
+        style: {
+          display: "none",
+        },
+        onChange: (e) => {
+          e.preventDefault();
+          if (multiple) {
+            onFormInputChange(config, e.target.files);
+          } else {
+            onFormInputChange(config, e.target.files[0]);
+          }
+        },
+      }, [
+        
+      ]),
+      label({
+        for: id,
+        className: 'form-file-label',
+      }, [
+        h(PublishIcon, {}),
+        uploadText,
+      ])
+    ])
+  ]);
```

Comment:
I am not 100% sure how file uploads work, especially since I don't have a real endpoint to test / figure out how it works on. Is this the right way of creating a generic `File` form field? Does it make sense for the form's value to be `e.target.files`?